test/puppetlabs/trapperkeeper/services/metrics/metrics_core_test.clj

```diff
@@ -1,23 +1,24 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-core-test
   (:import (com.codahale.metrics MetricRegistry JmxReporter))
   (:require [clojure.test :refer :all]
+            [puppetlabs.trapperkeeper.testutils.logging :refer :all]
             [puppetlabs.trapperkeeper.services.metrics.metrics-core :refer :all]
             [schema.test :as schema-test]))
 
 (use-fixtures :once schema-test/validate-schemas)
 
 (deftest test-initialize
-  (testing "does not initialize registry if metrics are disabled"
-    (let [context (initialize {:enabled false :server-id "localhost"})]
-      (is (nil? (:registry context)))))
-  (testing "initializes registry and adds to context if metrics are enabled"
-    (let [context (initialize {:enabled true :server-id "localhost"})]
+  (testing "it logs if :enabled is provided"
+    (with-test-logging
+      (let [context (initialize {:server-id "localhost" :enabled false})]
+        (is (logged? #"^Metrics are now always enabled." :warn)))))
```

Comment:
I'm thinking it might be good to add a test assertion here to ensure that the metrics registry is created when `:enabled` is set to `false`.